base/linking.jl

```diff
@@ -64,7 +64,7 @@ end
 
 function __init_dsymutil_path()
     #Same as with lld but for dsymutil
-    for bundled_dsymutil_path in (joinpath(Sys.BINDIR, Base.LIBEXECDIR, dsymutil_exe),
+    for bundled_dsymutil_path in (joinpath(Sys.BINDIR, Base.PRIVATE_LIBEXECDIR, dsymutil_exe),
```

Comment:
Move `dsymutil` too, or not modify this.
https://github.com/JuliaLang/julia/blob/931660663fb470e355b4e020bb604ab11e070e47/Makefile#L340-L341

Reply:
Thanks. I had worked on 1.9.0-beta4 initially and I hadn't noticed this one has been added since that release.